molecule/oauth-archive-plain-ubuntu2004/molecule.yml

```diff
@@ -117,6 +140,7 @@ provisioner:
   inventory:
```

Comment:
looks like```
 playbooks:
    converge: ${MIGRATION_CONVERGE:-../collections_converge.yml}
```
is missing here, the scenario won't work in zk to kraft migration, please add that as well